com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -216,6 +216,10 @@ public void testTf2()
     // once https://github.com/wala/ML/issues/106 is fixed.
     testTf2("tf2_test_model_call3.py", "SequentialModel.call", 1, 4, 2);
     testTf2("tf2_test_model_call4.py", "SequentialModel.__call__", 1, 4, 2);
+    testTf2("tf2_test_callbacks.py", "get_dataset", 0, 0);
```

Comment:
We need tensors (not 0, 0) in these tests. That is why we need the callbacks. If there's no tensor parameter, it's OK to at least have tensor variables.